modules/react/pagination/stories/PropTables.splitprops.tsx

```diff
@@ -0,0 +1,47 @@
+import {
+  PaginationModel,
+  PaginationState,
+  PaginationEvents,
+  UsePaginationModelConfig,
+} from '@workday/canvas-kit-react/pagination';
+
+// <ArgsTable of={Pagination} /> generates a very large props table given that
+// PaginationProps includes FlexProps. Use this dummy component instead to
+// limit the props shown.
+export const PaginationHoistedComponent = (_: {model: PaginationModel}) => <div />;
+
+// <ArgsTable of={Pagination.PageButton} /> generates a props table with
+// IconButton props. Use this dummy component instead to limit the props shown.
+export const PageButtonComponent = (_: {pageNumber: number}) => <div />;
```

Comment:
I had to add a few dummy components to [`PropTables.splitprops.tsx`](https://github.com/Workday/canvas-kit/pull/1295/files#diff-4ff84213be099deb3d4d749878f1a4e74de5dadfc0887759a9830e5ff83b87a0) in order to keep their prop tables at a manageable size. Let me know if you can think of a better way to handle this.